### ISSUE TITLE
Many RSVP "Defined in" links are broken for 2.18 and 3.0 API docs

### ISSUE THREAD

**Issue by BillyRayPreachersSon** [ember-learn/ember-api-docs#487]
On the pages https://emberjs.com/api/ember/2.18/functions/rsvp/race and https://emberjs.com/api/ember/3.0/functions/rsvp/race, the "Defined in" links are broken.

They currently point to https://github.com/emberjs/ember.js/tree/v2.18.2/node_modules/rsvp/lib/rsvp/race.js#L3 and https://github.com/emberjs/ember.js/tree/v3.0.0/node_modules/rsvp/lib/rsvp/race.js#L3, respectively, but these give 404s when visited.

It's not just the `race.js` source links that are broken... other `RSVP` source links also give 404s when visited.

The only reason I wanted to find the source is that there's no documentation about the method at all, other that saying that it's "a convenient alias for `RSVP.Promise.race` - and yet if I go to the `RSVP.Promise` docs (at https://emberjs.com/api/ember/2.18/classes/Promise or https://emberjs.com/api/ember/3.0/classes/Promise respectively), I see no methods defined other than `catch`, `finally`, and `then`. The "Defined in" links on those two pages are also broken.

**Comment by toddjordan** [comment]
Good catch and thanks for reporting.  we do some hackery since rsvp is in an external library, but there's a better way to do this that will address your issues.

**Comment by jscharf** [comment]
The "Defined in" link is also broken for Ember 2.12 on the following RSVP.Promise docs page:
https://www.emberjs.com/api/ember/2.12/classes/RSVP.Promise
